weblate_web/static/style-rtl.css

```diff
@@ -4032,71 +4033,501 @@ input.fullwidth {
+  min-height: 50px !important;
+}
+.med4 {
+  min-width: 150px !important;
+  min-height: 50px !important;
+}
+
+.med5 {
+  min-width: 170px !important;
+  min-height: 50px !important;
+}
+
+/* Services CSS (More Weblate Grid) */
+
+.inner-grid {
+  display: grid;
+  grid-template-columns: auto auto auto;
+  grid-column-gap: 0px;
+  grid-row-gap: 25px;
+}
+
+.grid-item2 p {
+  color: #2e344b;
+  font-size: 18px;
+  font-weight: 600;
+  letter-spacing: 0;
+  line-height: 30px;
+}
+
+.grid-item2 t {
+  color: #2e354a;
+  font-size: 16px;
+  letter-spacing: 0;
+  line-height: 10px;
+}
+
+.grid-item1 {
+  padding-top: 5px;
+}
+
+.grid-item2 {
+  padding-right: 15px;
+}
+
+.grid-item3 {
+  padding-top: 5px;
+}
+
+/* Services CSS (Inputs and placeholders) */
+
+input[type="email"] {
+  height: 50px;
+  width: 70%;
+  border-radius: 5px 5px 0 0;
+  background-color: #e9eaec;
+  border-bottom: 2px solid #c4c6cb;
+  border-top: none;
+  border-right: none;
+  border-left: none;
+  padding: 13px 5px;
+}
+
+::placeholder {
+  color: #bfc3c7;
+  font-size: 18px;
+  letter-spacing: 0;
+  line-height: 23px;
+  padding-right: 10px;
+}
+
+input.fullwidth {
+  box-sizing: border-box;
+  height: 50px;
+  width: 100%;
+  border: 2px solid #bfc3c7;
+  border-radius: 5px;
+}
+
+/* Services CSS (Users section) */
+
+.user-slot {
+  height: 60px;
+  width: 330px;
+  border: 1px solid #00d2e6;
+  border-radius: 30px;
+  margin-bottom: 10px;
+  opacity: 0.8;
+  position: relative;
+}
+
+.user-slot-content p {
+  color: #1fa385;
+  font-size: 14px;
+  letter-spacing: 0.86px;
+  line-height: 0px !important;
+  padding-bottom: 3px;
+}
+
+.user-slot-content t {
+  color: #2a3744;
+  font-size: 17px;
+  letter-spacing: 0;
+  line-height: 0px !important;
+}
+
+.user-img {
+  position: relative;
+  top: -14px;
+  right: 288px;
+}
+
+.user-slot-content {
+  position: relative;
+  top: 19px;
+  right: 30px;
+}
+
+/* Services CSS (Payments Section) */
+
+.payment-listing {
+  border-bottom: 1px solid #e9eaec;
+}
+
+.payment-listing:last-child {
+  border-bottom: none;
+}
+
+/* Services CSS (Services) */
+
+.service-variant {
+  display: flex;
+  justify-content: space-between;
+  height: 120px;
+  align-items: center;
+}
+.service-variant p {
+  color: #2b3646;
+  font-size: 40px;
+  letter-spacing: -1px;
+  line-height: 44px;
+  padding-right: 40px;
+  padding-top: 10px;
 }
 
-.weblate-team-content {
-  display: table-cell;
-  padding-bottom: 90px;
+.service-time {
+  display: grid;
+  text-align: center;
   padding-top: 20px;
+  padding-left: 40px;
+  color: #bfc3c7;
+  font-size: 17px;
 }
 
-.weblate-team {
-  display: table;
-  width: 82%;
+/* Services CSS (Custom paddings, styles and margins) */
+
+.bdr-btm {
+  border-bottom: none !important;
+}
+
+.style {
+  padding-top: 0px !important;
+  padding-bottom: 0px !important;
+}
+
+.pt-pb {
+  padding-top: 0px !important;
+  padding-bottom: 0px !important;
+}
+
+.pt-1 {
+  padding-top: 22px !important;
+}
+
+.style6 > div {
```

Comment:
Can we please have reasonable class names? "style6" really doesn't tell anything on how it is being used. Somebody will look at your code few years later and should be able to understand it and having descriptive names really helps in this.

Reply:
Yes, you are right. I apologize for this, i will change it.